snuba/clickhouse/columns.py

```diff
@@ -69,7 +80,7 @@ def for_schema(self) -> str:
     def flatten(self, name: str) -> Sequence[FlattenedColumn]:
         return [FlattenedColumn(None, name, self)]
 
-    def get_all_modifiers(self) -> Iterable[Type[ColumnTypeWithModifier]]:
+    def get_all_modifiers(self) -> List[Type[ColumnTypeWithModifier]]:
```

Comment:
Same thing as above about `List` v. `MutableSequence`.

Reply:
This one can be fixed instead.